app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPickerViewModel.kt

```diff
@@ -50,11 +50,9 @@ class AvatarPickerViewModel @Inject constructor(
     fun uploadNewPickedAvatarAndBack(imgUri: Uri, context: Context) {
         viewModelScope.launch {
             withContext(Dispatchers.IO) {
+                postProcessCapturedAvatar(imgUri, context)
                 val data = imgUri.toByteArray(context)
-                val extension = MimeTypeMap.getFileExtensionFromUrl(imgUri.path)
-                val mimeType = context.contentResolver.getType(imgUri)
-                    ?: MimeTypeMap.getSingleton().getMimeTypeFromExtension(extension)
-                    ?: "image/$extension"
+                val mimeType = imgUri.getMimeType(context) ?: "image/jpg"
```

Comment:
I believe it should be `image/jpeg`: https://www.w3.org/Graphics/JPEG/
`image/jpg` should also work in general, but if we can I think we should stick to the first one

Reply:
Thanks, totally agree, we should stick to the standard !